.vscode/tasks.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Is this task used for anything? It looks like the `build.sh` file relies on something in `/opt/carma`, but it's unclear why it needs to be  VS COde task.